### ISSUE TITLE
Add support for 7-segment displays

### ISSUE THREAD

**Issue by jabdoa2** [missionpinball/mpf#408]
1. Add support in hardware platform
2. Add 7-segment device
3. Implement some alphanumeric_display_config_player to use it in shows

From: https://groups.google.com/d/msg/mpf-users/IRWEfd1Dbjk/a3_kHx6pAQAJ

**Comment by toomanybrians** [comment]
I don't believe that any of the pinball hardware players support 7-segment displays at this time. So any support we do will need to be a bit more homebrew-ish with some kind of USB-to-segment controllers or something.

However we do this, we should also think about doing the 15-segment alpha-numeric displays at the same time. I believe that FAST currently supports the WPC-era alpha-numerics (which were alpha segment displays driven by the 14-pin DMD connector in early WPC alpha games like BOP and Funhouse), but I don't believe FAST has a board out for earlier solutions.

Jim Askey has some alpha-numeric interface boards that work with Snux's System 11 board, and I know the DIY Pinball guys will have it too.

Bottom line though is I think we're sort of waiting for more hardware support for this to become more popular, though of course we could start building it sooner. I would imagine we can build the device once we figure out the interface to the hardware platform modules, and then we can just add more hardware platforms as they come along.

**Comment by jabdoa2** [comment]
done in dev
